### DISCUSSION TITLE
Where is MathBase class?

### DISCUSSION THREAD

**Discussion by edumacs** [GSI-CS-CO/chart-fx#408] [Q&A]
I just tried to test the code but I got the MathBase class missing.
Could you help me, please. Thanks a lot.

**Comment by wirew0rm** [comment]
Try building with maven on the command line. Some math classes are autogenerated from templates during the build and e.g. intellij-idea does not call the relevant plugin. Subsequent builds from the IDE will also work, provided that you either do not do a clean or install the chartfx-generate plugin into your local maven repository (mvn install from command line).
